src/Modal/Modal.jsx

```diff
@@ -67,10 +67,10 @@ class Modal extends Component {
       >
         <ModalStyles>
           <Overlay onClick={onClose} />
-          <Height>
-            <Row position="center" verticalAlign="middle">
-              <Col lg={size === "sm" ? 7 : 12} sm={12}>
-                <Content>
+          <Row position="center" verticalAlign="middle">
+            <Col lg={size === "sm" ? 7 : 12} sm={12}>
+              <Content>
+                <HeaderArea>
```

Comment:
also not convinced that we need to add this?

Reply:
yeah you are right in the beginning I didnt add it as well and in the browser it looks great, when the tests run though the modal doesnt look as it supposed to be looking and adding `flex: 1 0` will give the right space to the header and the bottom. 

wanna talk about it in person maybe we find a better solution?